rosidl_typesupport_opensplice_cpp/bin/rosidl_typesupport_opensplice_cpp

```diff
@@ -8,6 +9,15 @@ from rosidl_typesupport_opensplice_cpp import generate_dds_opensplice_cpp
 from rosidl_typesupport_opensplice_cpp import generate_typesupport_opensplice_cpp
 
 
+def is_valid_file(parser, file_name):
+    if not os.path.exists(file_name):
+        parser.error("File does not exist: '{0}'".format(file_name))
+    file_name = os.path.abspath(file_name)
```

Comment:
At the very least it's slightly clearer for the following error message, but I can avoid mutating the input. Either way it shouldn't matter, `abspath` won't work if the relative path wouldn't also work.